File: AvailabilityCollector/Models/Worker.cs
namespace AvailabilityCollector.Models;

public class Worker
{
    public int ID { get; set; }
    public string Ime { get; set; } = "";
    public string Priimek { get; set; } = "";
    public string DelovnoMesto { get; set; } = "";
    public string VrstaZaposlitve { get; set; } = ""; 
    public DateTime CreatedAt { get; set; } = DateTime.Now;
    public bool IsActive { get; set; } = true;

    // Navigation
    public ICollection<Razpolozljivost>? Razpolozljivosti { get; set; }
    public ICollection<Status>? Statusi { get; set; }
}

File: Android/app/src/main/java/Android/availibityCollector/data/models/Worker.kt
package Android.availibityCollector.data.models

import com.google.gson.annotations.SerializedName

data class Worker(
    @SerializedName("id") val id: Int,
    @SerializedName("ime") val ime: String,
    @SerializedName("priimek") val priimek: String,
    @SerializedName("delovnoMesto") val delovnoMesto: String,
    @SerializedName("vrstaZaposlitve") val vrstaZaposlitve: String,
    @SerializedName("createdAt") val createdAt: String,
    @SerializedName("isActive") val isActive: Boolean
)
